core/service/subscription.go

```diff
@@ -35,23 +37,23 @@ import (
 	"github.com/rs/zerolog/log"
 )
 
-func subscribeSessionCreate(mng *session.Manager, ch p2p.Channel, service Service, eventPublisher eventbus.Publisher) {
+func subscribeSessionCreate(mng *session.Manager, ch p2p.Channel, service Service, eventPublisher eventbus.Publisher, proposal market.ServiceProposal, policyRules *policy.Repository) {
```

Comment:
You dont need to pass new arguments, running service instance `service Service` contains those

Reply:
`service Service` is an interface that allows us to `Serve` `Stop` and `ProvideConfig`. These arguments required outside the service instance.